processor/redactionprocessor/README.md

```diff
@@ -60,6 +60,10 @@ processors:
       - group
       - id
       - name
+    # Ignore the following attributes, allow them to pass without redaction.
```

Comment:
Could the comment for `allowed_keys` above be updated to not say "pass through", but instead something like:

> allowed_keys is a list of span attribute keys that will be kept on the span and processed.

Maybe also remove the following sentence:

> To allow the span attributes you know are good, add them to the list.